cvs diff: Diffing sw
cvs diff: Diffing sw/inc
cvs diff: Diffing sw/inc/pch
cvs diff: Diffing sw/prj
cvs diff: Diffing sw/qa
cvs diff: Diffing sw/qa/complex
cvs diff: Diffing sw/qa/complex/accessibility
cvs diff: Diffing sw/qa/complex/checkColor
cvs diff: Diffing sw/qa/complex/indeterminateState
cvs diff: Diffing sw/qa/complex/writer
cvs diff: Diffing sw/qa/core
cvs diff: Diffing sw/qa/unoapi
cvs diff: Diffing sw/qa/unoapi/testdocuments
cvs diff: Diffing sw/sdi
cvs diff: Diffing sw/source
cvs diff: Diffing sw/source/core
cvs diff: Diffing sw/source/core/SwNumberTree
cvs diff: Diffing sw/source/core/access
cvs diff: Diffing sw/source/core/attr
cvs diff: Diffing sw/source/core/bastyp
cvs diff: Diffing sw/source/core/crsr
cvs diff: Diffing sw/source/core/doc
cvs diff: Diffing sw/source/core/docnode
cvs diff: Diffing sw/source/core/draw
cvs diff: Diffing sw/source/core/edit
cvs diff: Diffing sw/source/core/except
cvs diff: Diffing sw/source/core/fields
cvs diff: Diffing sw/source/core/frmedt
cvs diff: Diffing sw/source/core/graphic
cvs diff: Diffing sw/source/core/inc
cvs diff: Diffing sw/source/core/layout
cvs diff: Diffing sw/source/core/objectpositioning
cvs diff: Diffing sw/source/core/ole
cvs diff: Diffing sw/source/core/para
cvs diff: Diffing sw/source/core/sw3io
cvs diff: Diffing sw/source/core/swg
cvs diff: Diffing sw/source/core/table
cvs diff: Diffing sw/source/core/text
cvs diff: Diffing sw/source/core/tox
cvs diff: Diffing sw/source/core/txtnode
cvs diff: Diffing sw/source/core/undo
cvs diff: Diffing sw/source/core/unocore
cvs diff: Diffing sw/source/core/view
cvs diff: Diffing sw/source/filter
cvs diff: Diffing sw/source/filter/ascii
cvs diff: Diffing sw/source/filter/basflt
cvs diff: Diffing sw/source/filter/html
cvs diff: Diffing sw/source/filter/inc
cvs diff: Diffing sw/source/filter/rtf
cvs diff: Diffing sw/source/filter/writer
cvs diff: Diffing sw/source/filter/ww1
cvs diff: Diffing sw/source/filter/ww8
cvs diff: Diffing sw/source/filter/ww8/dump
cvs diff: Diffing sw/source/filter/xml
cvs diff: Diffing sw/source/ui
cvs diff: Diffing sw/source/ui/app
cvs diff: Diffing sw/source/ui/cctrl
cvs diff: Diffing sw/source/ui/chrdlg
cvs diff: Diffing sw/source/ui/config
cvs diff: Diffing sw/source/ui/dbui
cvs diff: Diffing sw/source/ui/dialog
Index: sw/source/ui/dialog/makefile.mk
===================================================================
RCS file: /cvs/sw/sw/source/ui/dialog/makefile.mk,v
retrieving revision 1.19
retrieving revision 1.19.186.1
diff -p -u -u -p -b -w -B -r1.19 -r1.19.186.1
--- sw/source/ui/dialog/makefile.mk	27 Sep 2007 11:37:21 -0000	1.19
+++ sw/source/ui/dialog/makefile.mk	12 Feb 2008 12:42:46 -0000	1.19.186.1
@@ -46,6 +46,10 @@ LIBTARGET=no
 .INCLUDE :  settings.mk
 .INCLUDE :  $(PRJ)$/inc$/sw.mk
 
+.IF "$(ENABLE_LAYOUT)" == "TRUE"
+CFLAGS+= -DENABLE_LAYOUT=1 -I../$(PRJ)/layout/inc -I../$(PRJ)/layout/$(INPATH)/inc
+.ENDIF # ENABLE_LAYOUT == TRUE
+
 # --- Files --------------------------------------------------------
 
 SRS1NAME=$(TARGET)
Index: sw/source/ui/dialog/wordcountdialog.cxx
===================================================================
RCS file: /cvs/sw/sw/source/ui/dialog/wordcountdialog.cxx,v
retrieving revision 1.8
retrieving revision 1.8.186.1
diff -p -u -u -p -b -w -B -r1.8 -r1.8.186.1
--- sw/source/ui/dialog/wordcountdialog.cxx	27 Sep 2007 11:39:08 -0000	1.8
+++ sw/source/ui/dialog/wordcountdialog.cxx	12 Feb 2008 12:42:46 -0000	1.8.186.1
@@ -50,6 +50,7 @@
 #endif
 
 #include <dialog.hrc>
+#include <layout/layout-pre.hxx>
 #include <wordcountdialog.hrc>
 
 /*-- 06.04.2004 16:05:55---------------------------------------------------
cvs diff: Diffing sw/source/ui/dochdl
cvs diff: Diffing sw/source/ui/docvw
cvs diff: Diffing sw/source/ui/envelp
cvs diff: Diffing sw/source/ui/fldui
cvs diff: Diffing sw/source/ui/fmtui
cvs diff: Diffing sw/source/ui/frmdlg
cvs diff: Diffing sw/source/ui/globdoc
cvs diff: Diffing sw/source/ui/inc
Index: sw/source/ui/inc/wordcountdialog.hxx
===================================================================
RCS file: /cvs/sw/sw/source/ui/inc/wordcountdialog.hxx,v
retrieving revision 1.3
retrieving revision 1.3.1148.1
diff -p -u -u -p -b -w -B -r1.3 -r1.3.1148.1
--- sw/source/ui/inc/wordcountdialog.hxx	9 Sep 2005 10:18:42 -0000	1.3
+++ sw/source/ui/inc/wordcountdialog.hxx	12 Feb 2008 12:42:47 -0000	1.3.1148.1
@@ -43,6 +43,8 @@
 #ifndef _SV_BUTTON_HXX
 #include <vcl/button.hxx>
 #endif
+#include <layout/layout.hxx>
+#include <layout/layout-pre.hxx>
 struct SwDocStat;
 class SwWordCountDialog : public SfxModalDialog
 {
@@ -70,4 +72,6 @@ public:    
     void    SetValues(const SwDocStat& rCurrent, const SwDocStat& rDoc);
 };
 
+#include <layout/layout-post.hxx>
+
 #endif
cvs diff: Diffing sw/source/ui/index
cvs diff: Diffing sw/source/ui/lingu
cvs diff: Diffing sw/source/ui/misc
cvs diff: Diffing sw/source/ui/ribbar
cvs diff: Diffing sw/source/ui/shells
cvs diff: Diffing sw/source/ui/smartmenu
cvs diff: Diffing sw/source/ui/table
cvs diff: Diffing sw/source/ui/uiview
cvs diff: Diffing sw/source/ui/uno
cvs diff: Diffing sw/source/ui/utlui
cvs diff: Diffing sw/source/ui/web
cvs diff: Diffing sw/source/ui/wrtsh
cvs diff: Diffing sw/uiconfig
cvs diff: Diffing sw/uiconfig/sglobal
cvs diff: Diffing sw/uiconfig/sglobal/accelerator
cvs diff: Diffing sw/uiconfig/sglobal/accelerator/de
cvs diff: Diffing sw/uiconfig/sglobal/accelerator/en-GB
cvs diff: Diffing sw/uiconfig/sglobal/accelerator/en-US
cvs diff: Diffing sw/uiconfig/sglobal/accelerator/es
cvs diff: Diffing sw/uiconfig/sglobal/accelerator/fr
cvs diff: Diffing sw/uiconfig/sglobal/menubar
cvs diff: Diffing sw/uiconfig/sglobal/statusbar
cvs diff: Diffing sw/uiconfig/sglobal/toolbar
cvs diff: Diffing sw/uiconfig/sweb
cvs diff: Diffing sw/uiconfig/sweb/accelerator
cvs diff: Diffing sw/uiconfig/sweb/accelerator/de
cvs diff: Diffing sw/uiconfig/sweb/accelerator/en-GB
cvs diff: Diffing sw/uiconfig/sweb/accelerator/en-US
cvs diff: Diffing sw/uiconfig/sweb/accelerator/es
cvs diff: Diffing sw/uiconfig/sweb/accelerator/fr
cvs diff: Diffing sw/uiconfig/sweb/menubar
cvs diff: Diffing sw/uiconfig/sweb/statusbar
cvs diff: Diffing sw/uiconfig/sweb/toolbar
cvs diff: Diffing sw/uiconfig/swform
cvs diff: Diffing sw/uiconfig/swform/accelerator
cvs diff: Diffing sw/uiconfig/swform/accelerator/de
cvs diff: Diffing sw/uiconfig/swform/accelerator/en-US
cvs diff: Diffing sw/uiconfig/swform/accelerator/es
cvs diff: Diffing sw/uiconfig/swform/accelerator/fr
cvs diff: Diffing sw/uiconfig/swform/menubar
cvs diff: Diffing sw/uiconfig/swform/statusbar
cvs diff: Diffing sw/uiconfig/swform/toolbar
cvs diff: Diffing sw/uiconfig/swreport
cvs diff: Diffing sw/uiconfig/swreport/accelerator
cvs diff: Diffing sw/uiconfig/swreport/accelerator/de
cvs diff: Diffing sw/uiconfig/swreport/accelerator/en-GB
cvs diff: Diffing sw/uiconfig/swreport/accelerator/en-US
cvs diff: Diffing sw/uiconfig/swreport/accelerator/es
cvs diff: Diffing sw/uiconfig/swreport/accelerator/fr
cvs diff: Diffing sw/uiconfig/swreport/menubar
cvs diff: Diffing sw/uiconfig/swreport/statusbar
cvs diff: Diffing sw/uiconfig/swreport/toolbar
cvs diff: Diffing sw/uiconfig/swriter
cvs diff: Diffing sw/uiconfig/swriter/accelerator
cvs diff: Diffing sw/uiconfig/swriter/accelerator/de
cvs diff: Diffing sw/uiconfig/swriter/accelerator/en-GB
cvs diff: Diffing sw/uiconfig/swriter/accelerator/en-US
cvs diff: Diffing sw/uiconfig/swriter/accelerator/es
cvs diff: Diffing sw/uiconfig/swriter/accelerator/fr
cvs diff: Diffing sw/uiconfig/swriter/menubar
cvs diff: Diffing sw/uiconfig/swriter/statusbar
cvs diff: Diffing sw/uiconfig/swriter/toolbar
cvs diff: Diffing sw/uiconfig/swxform
cvs diff: Diffing sw/uiconfig/swxform/accelerator
cvs diff: Diffing sw/uiconfig/swxform/accelerator/de
cvs diff: Diffing sw/uiconfig/swxform/accelerator/en-GB
cvs diff: Diffing sw/uiconfig/swxform/accelerator/en-US
cvs diff: Diffing sw/uiconfig/swxform/accelerator/es
cvs diff: Diffing sw/uiconfig/swxform/accelerator/fr
cvs diff: Diffing sw/uiconfig/swxform/menubar
cvs diff: Diffing sw/uiconfig/swxform/statusbar
cvs diff: Diffing sw/uiconfig/swxform/toolbar
cvs diff: Diffing sw/util
Index: sw/util/makefile.mk
===================================================================
RCS file: /cvs/sw/sw/util/makefile.mk,v
retrieving revision 1.62
retrieving revision 1.62.164.1
diff -p -u -u -p -b -w -B -r1.62 -r1.62.164.1
--- sw/util/makefile.mk	15 Oct 2007 12:15:54 -0000	1.62
+++ sw/util/makefile.mk	12 Feb 2008 12:42:47 -0000	1.62.164.1
@@ -199,6 +199,11 @@ SHL3STDLIBS= \
             $(SALLIB) \
 			$(SOTLIB)
 
+.IF "$(ENABLE_LAYOUT)" == "TRUE"
+LINKFLAGS+= -L../$(PRJ)/layout/$(INPATH)/lib
+SHL3STDLIBS += -ltklayout$(UPD)$(DLLPOSTFIX)
+.ENDIF # ENABLE_LAYOUT == TRUE
+
 SHL3LIBS=   $(SLB)$/swui.lib
 LIB3TARGET = $(SLB)$/swui.lib
 
cvs diff: Diffing sw/util/defs
cvs diff: Diffing sw/xml
